plugins/blip/basics/less/sitechange/Change.less

```diff
@@ -6,6 +6,15 @@
   @font-size: 14px;
   @line-height: 16px;
 
+  .Change-count-text {
+    display: block;
+    position: absolute;
+    font-size: 12px;
+    left: @infusion-line-start-x - ((@infusion-line-mark-size/2) + 1);
+    top: @infusion-line-y-base - 22px;
```

Comment:
Any chance we can define the 22px less magically? Does it make sense to define it in terms of the font's line height for example?